_includes/master/reqs-sys.md

```diff
@@ -0,0 +1,44 @@
+## Node requirements
+
+- AMD64 processor
+
+- Linux kernel 3.10 or later. Kernel dependencies: `nf_conntrack_netlink` subsystem, 
```

Comment:
You lost the bit of information about IPv4 only having fewer requirements (e.g. IPv4 only requires kernel: 2.6.32)

Reply:
Yes, but I was reading in the Felix binary install page that if you intend to use containers, you need 3.1.0 or later (a Docker req). So.....it seems like that boils down to: you need 3.1.0 or later.